src/Components/Input/index.jsx

```diff
@@ -0,0 +1,42 @@
+// Dependencies
+import React from "react";
+import { useState } from 'react'
+// Components
+
+// Styles
+import { StyledInput } from "./styles";
+import IconSVG from 'Components/Icon/IconSVG'
+
+// handle prop renders the placeholder for certain input
+
+const Input = ({ type, handle }) => {
```

Comment:
Are you sure it will just be these props? I think they could be some more for example: disabled, maxLength, defaultValue,  onChange, etc...

on the other hand, why call **handle** to the prop that will be used in the placeholder?